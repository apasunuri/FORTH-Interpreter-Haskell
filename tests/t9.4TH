10 10 * 7 4 * + EMIT