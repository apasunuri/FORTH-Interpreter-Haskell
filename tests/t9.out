 FORTH.exe: Integer Out of Range